demos/tests/cases.py

```diff
@@ -371,33 +371,49 @@ def single_option_cases(key, *args):
             ModelArg('person-reidentification-retail-0288')),
     )),
 
-    PythonDemo(subdirectory='object_detection_demo_ssd_async', device_keys=['-d'], test_cases=combine_cases(
-        TestCase(options={'--no_show': None,
-            **MONITORS,
-            '-i': DataPatternArg('object-detection-demo-ssd-async')}),
-        single_option_cases('-m',
-            ModelArg('face-detection-adas-0001'),
-            ModelArg('face-detection-retail-0004'),
-            ModelArg('face-detection-retail-0005'),
-            ModelArg('face-detection-retail-0044'),
-            ModelArg('pedestrian-and-vehicle-detector-adas-0001'),
-            ModelArg('pedestrian-detection-adas-0002'),
-            ModelArg('person-detection-retail-0013'),
-            ModelArg('vehicle-detection-adas-0002'),
-            ModelArg('vehicle-license-plate-detection-barrier-0106'),
-            ModelArg('ssd-resnet34-1200-onnx')),
-    )),
-
-    PythonDemo(subdirectory='object_detection_demo_yolov3_async', device_keys=['-d'], test_cases=combine_cases(
-        TestCase(options={'--no_show': None,
-            **MONITORS,
-            '-i': DataPatternArg('object-detection-demo-ssd-async')}),
-        single_option_cases('-m',
-            ModelArg('yolo-v1-tiny-tf'),
-            ModelArg('yolo-v2-tiny-tf'),
-            ModelArg('yolo-v2-tf'),
-            ModelArg('yolo-v3-tf'),
-            ModelArg('mobilefacedet-v1-mxnet')),
+    PythonDemo(subdirectory='object_detection_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'--no_show': None, **MONITORS, '-i': DataPatternArg('object-detection-demo-ssd-async')}),
+        [
+            *combine_cases(
+                TestCase(options={'--architecture_type': 'ssd'}),
```

Comment:
Please arrange the types in alphabetical order.

Reply:
fixed